frontend/src/component/feature/StrategyTypes/StrategyVariants.tsx

```diff
@@ -54,7 +55,7 @@ export const StrategyVariants: FC<{
         setVariantsEdit(
             (strategy.variants || []).map((variant) => ({
                 ...variant,
-                new: false,
+                new: editable || false,
```

Comment:
This is the important change that allows us to edit the name if editable is passed in